elastic/kibana
# [XY axis] Updates the sample data to work with the default eui palette

## Summary

Closes #94067. This PR updates our XY axis sample viz in order to use the default EUI palette instead of the legacy one. I have added the palette configuration on the saved objects and I have updated the migration script to only add the kibana legacy palette configuration for the saved objects that don't contain this info.

The reason that we are doing this is because we want to drive our users towards the EUI palettes.

### How to test
Install the sample datasets again and check all the XY axis visualizations (the Legacy charts advanced setting should be off). They should load with the EUI palette, for example:
![image](https://user-images.githubusercontent.com/17003240/111442368-f4219480-8710-11eb-9f17-1031360555b5.png)

Set the switch to on (enable the vislib charts). The visualizations should render with the legacy palette.

### Checklist
